janusgraph-es/src/test/java/org/janusgraph/diskstorage/es/CQLElasticsearchTest.java

```diff
@@ -59,4 +57,8 @@ protected boolean supportsCollections() {
         return true;
     }
 
+    @Override
+    @Disabled("CQL seems to not clear storage correctly")
```

Comment:
Do we have a bug with CQL so that it cannot clear storage right now or is it a normal behavior? Should we open an issue to track it?

Reply:
It seems to be an issue which only exists in combination with elasticsearch. Could be a timing problem of the test? We should open an issue. I think clearing should not be a blocker which basically only used in tests.